utils-intervalrdd/src/main/scala/org/apache/spark/intervalrdd/IntervalRDD.scala

```diff
@@ -0,0 +1,184 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bdgenomics.utils.intervalrdd
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.storage.StorageLevel
+import org.apache.spark.{ Logging, Partition, TaskContext, _ }
```

Comment:
This could be a problem.  By conflicts do you mean that it won't work?  `org.apache.spark.Logging` was made private and now has [moved to a new package](https://github.com/apache/spark/commit/8ef3399aff04bf8b7ab294c0f55bcf195995842b).